### PR TITLE
Update yamls for TN State Sen Republicans (VAN-50740) and MI State Sen O'Brien (VAN-50747) + create yaml for fish&wildlife director (VAN-50723) + update MD state yamls w/ JS step (VAN-50276) 

### PR DESCRIPTION
**Pull request by OdeclasV**
**(VAN-50740)**: yamls for TN state senate republicans were failing due to outdated CSS selectors. I changed all of the yamls we have for them, so there are quite a few changes in this PR related to this ticket. 

**(VAN-50747)**: MI sate senator Margaret O'Brien's yaml was also failing due to changes in the selectors. I updated them but there seems to be some issues with the target form that is preventing the messages to go through. Tom and I did several tests to no avail. I'll add more info about this in the JIRA ticket. Changes for this yaml, though, should be added since the selectors changed.

**(VAN-50723)**: Creating new yaml for fish and wildlife director as he was added in Artemis as a cabinet member

**(VAN-50276)**: Fixing 11-digit phone number issue in these yamls by adding a javascript step.

### CODE REVIEW

File: members/TNL000007.yaml [hunk 1]
```diff
@@ -1,105 +1,54 @@
...
-          - Sen. Mae Beavers
-          - Sen. Mark Green
-          - Sen. Mark Norris
-          - Sen. Mike Bell
-          - Sen. Randy McNally
-          - Sen. Rusty Crowe
-          - Sen. Steve Southerland
-          - Sen. Steven Dickerson
-          - Sen. Todd Gardenhire
+          Sen. Rusty Crowe
```
Comment (annanhan) [inline members/TNL000007.yaml]: does this need to be `Sen. Rusty Crowe : Sen. Rusty Crow`?

File: members/TNL000007.yaml [hunk 2]
```diff
@@ -1,105 +1,54 @@
...
+    - visit: "http://www.tngopsenate.com/contact"
     - fill_in:
-        - name: fname
-          selector: "#fscf_f_name1"
-          value: $NAME_FIRST
-          required: true
-        - name: lname
-          selector: "#fscf_l_name1"
-          value: $NAME_LAST
+        - name: firstname
```
Comment (annanhan) [inline members/TNL000007.yaml]: you should call these fullname

**Comment by annanhan** [comment]
Will the MD YAMLs still work with the indention changes?

**Comment by OdeclasV** [comment]
@annanhan yes, MD yamls still work with the indentation. The original indentation was off by one tab to right.

**Pull request merged**